### ISSUE TITLE
Add doesn't support Custom Secure Notes templates

### ISSUE THREAD

**Issue by speshak** [lastpass/lastpass-cli#190]
`lpass add` doesn't support using custom note template types.  I have an "Ansible Vault" type defined in my account:

lpass add --note-type="Ansible Vault"                                              
Usage: lpass ... --note-type=TYPE

Valid values for TYPE:
	amex
	bank
	credit-card
	database
	drivers-license
	email
	health-insurance
	im
	insurance
	mastercard
	membership
	passport
	server
	software-license
	ssh-key
	ssn
	visa
	wifi

**Comment by bcopeland** [comment]
Indeed, we only support the hard-coded ones at this time.  Will look into what's needed for custom templates.

**Comment by glennmcallister** [comment]
Any progress on this?

**Comment by bcopeland** [comment]
On Mon, Jun 05, 2017 at 02:25:57PM -0700, Glenn McAllister wrote:
> Any progress on this?

None yet.

**Comment by MartinSohn** [comment]
Any progress on this issue?

Until this feature is implemented it is very hard to automate IT systems using Lastpass.

**Comment by nedbase-jesse** [comment]
@bcopeland Any progress on this issue / feature?
We require custom types of notes to migrate credentials and information to LastPass

**Comment by ffananas-cs** [comment]
This is a key item for automation, it would be very useful to have this feature.

**Comment by jimmythedog** [comment]
Please implement this feature!
Automation is extremely difficult without this

**Comment by ffananas-cs** [comment]
Yes it would be very good to know if there are plans to implement, if not then we would have to look at an alternative to using that to be able to automate, some feedback would be appreciated.

**Comment by JG127** [comment]
Same for me. For the sake of automation too.

**Comment by supernomad** [comment]
This issue has been open for over three years, and from what I can tell there is no movement at all on the subject. If the plan is to not implement this particular feature can the maintainers say so and close this issue?

**Comment by gannas42** [comment]
Looking for an update on whether this will be pursued.

**Comment by itfranck** [comment]
I stumbled upon this problem myself while trying to add a custom note. 
Looks like there's nothing I can do for now :(

**Comment by pccowboy** [comment]
I'll pile on, any thoughts about whether this will be supported?

**Comment by blami** [comment]
Hi any idea when this will be supported? I'd like to upload custom notes too. This is crucial for automation (e.g. cloud deployments).

**Comment by speshak** [comment]
It's been a little over 5 years since I first opened this issue, so I wouldn't expect anything soon. I personally have since stopped trying to use custom notes because that was an easier way to move forward.

**Comment by craigatfetch** [comment]
I feel at this point that the lpass CLI isn't an active project at LastPass.  I don't think any new features will be added.
